src/Mobs/Monster.cpp

```diff
@@ -883,7 +885,7 @@ void cMonster::InStateEscaping(std::chrono::milliseconds a_Dt, cChunk & a_Chunk)
 
 void cMonster::ResetAttackCooldown()
 {
-	m_AttackCoolDownTicksLeft = static_cast<int>(20 * m_AttackRate);  // A second has 20 ticks, an attack rate of 1 means 1 hit every second
+	m_AttackCoolDownTicksLeft = static_cast<int>(m_AttackRate);  // A second has 20 ticks, an attack rate of 1 means 1 hit every 3 seconds
```

Comment:
This looks like a bad change. Now the comment doesn't make any sense (why does it mention the 20 ticks?) and why is it 3 seconds again? Wasn't this changed recently?

Reply:
Just a debug change that shouldn't have made it in.